### ISSUE TITLE
Rumble support 

### ISSUE THREAD

**Issue by Toiletmason** [polymorphicshade/NewPipe#321]
Rumble is getting bigger these days, and competition is always good. The original project deva refuse due to political reasons so they refuse to support it. There is a project that has already built it to work, but they don't have the other features this fork has so it makes more sense to merge that piece into this project.

**Comment by Mr-Bajs** [comment]
Newpipe have said they wont support Rumble because reasons. There is a fork of newpipe https://github.com/bravenewpipe that supports additional sources, i think they support Rumble.

**Comment by Toiletmason** [comment]
@Mr-Bajs Yeah, I already know about that project I just didn't name it. There has been more things done with this fork so I do think it would be better to just merge that to this project. There are some things from animepipe too that would be nice like creator page tabs, and reading comment replys, but I feel like those are less impressive at the moment.

**Comment by Mr-Bajs** [comment]
https://github.com/polymorphicshade/NewPipe/issues/225

Answers this question. So i'll guess that if the devs of BraveNewpipe insted just made PR to this (or the extractor) then it would de fine. I don't know why they dont' though, maybe a missunderstanding?

**Comment by Mr-Bajs** [comment]
> @Mr-Bajs I don't know how you're even supposed to ask as they don't have an issues page for their project as far as I can tell.

I raised the question in an issue in the https://github.com/bravenewpipe/NewPipeExtractor/issues
That's where the development of the fork is i suppose hence they just add sources,

**Comment by polymorphicshade** [comment]
I have no plans on working on this, but I will welcome any pull requests that add this feature.
